### PR TITLE
Add overrideNextButtonText

### PR DESCRIPTION
**Pull request by eschramm**
Closes https://github.com/CareEvolution/RKStudio-Participant/issues/1134. UI Tests available in https://github.com/CareEvolution/CEVResearchKit/pull/274.

## Testing

See https://github.com/CareEvolution/CEVResearchKit/pull/274

### CODE REVIEW

File: ORK1Kit/ORK1Kit/Common/ORK1Step.h [hunk 1]
```diff
@@ -233,6 +233,13 @@ The default value of this property is 'NO'.
 */
 @property (nonatomic, assign) BOOL excludeFromProgressCalculation;
 
+/**
+An override title for the next/done button.
+
+The default value of this property is 'nil'.
+*/
+@property (nonatomic, copy, nullable) NSString *overrideNextButtonText;
```
Comment (mmertsock) [inline ORK1Kit/ORK1Kit/Common/ORK1Step.h]: Can this just be named `nextButtonText` instead?

**Review by mmertsock — APPROVED** [review]
Works well!

I thought you would need to change ORK1InstructionStepViewController, perhaps in `useAppropriateButtonTitleAsLastBeginningInstructionStep`, to handle the "Get Started" special case, but it appears that `setupButtons` in ORK1StepViewController is called after useAppropriateButtonTitleAsLastBeginningInstructionStep, so the button text does still get overridden. So there's a risk this could break in the future if the order of function calls changes somehow, but that seems unlikely.

File: ORK1Kit/ORK1Kit/Common/ORK1Step.h (hunk 1 above)
Comment (eschramm) [reply ORK1Kit/ORK1Kit/Common/ORK1Step.h]: 👍

Comment (mmertsock) [reply ORK1Kit/ORK1Kit/Common/ORK1Step.h]: Latest changes LGTM

**Pull request merged**